portfolio/.theia/launch.json

```diff
@@ -0,0 +1,8 @@
+{
+  // Use IntelliSense to learn about possible attributes.
+  // Hover to view descriptions of existing attributes.
```

Comment:
optional: remove unless you intentionally added this as a reminder to yourself?

Looks like Github is highlighting this text red because it is a syntax error - [stackoverflow](https://stackoverflow.com/questions/30667675/what-does-a-red-text-background-mean-in-github-comparison-view). JSONs syntax technically doesn't support comments (wat??). This is all news to me.

Whatever is parsing this is likely more forgiving.

Reply:
As far as I'm aware, I didn't make any changes to the launch.json file. I will go back and remove those lines though.